### ISSUE TITLE
IRC announce constantly generates error

### ISSUE THREAD

**Issue by Bevinsky** [R-a-dio/Hanyuu-sama#53]
The IRC announcing has been unruly for a long time now. It throws a really annoying error in the main manager console:

```
ERROR:root:IRC Announcing error
Traceback (most recent call last):
  File "/home/hanyuu/Hanyuu-sama/manager.py", line 1171, in change
    irc.connect().announce()
  File "<string>", line 2, in announce
  File "/usr/lib/python2.7/multiprocessing/managers.py", line 773, in _callmethod
    raise convert_to_error(kind, result)
RemoteError:
---------------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/managers.py", line 242, in serve_client
    obj, exposed, gettypeid = id_to_obj[ident]
KeyError: 'b52310'
---------------------------------------------------------------------------
```

This happens sometimes and sometimes not. Also, calling the method manually (often twice) somehow works, and also has the effect of "clearing the pipe" for the regular call.

This has been happening for a really long time now and it's high time it got fixed.

**Comment by ghost** [comment]
Are you using an SSH tunneled connection?

**Comment by Bevinsky** [comment]
We think it might have something to do with how the multiprocessing managers work. We are switching to a different kind in 1.3, so we should be rid of this error by then.

**Comment by ghost** [comment]
Yeah, that was my first thought. Switching should fix it right up.

**Comment by Wessie** [comment]
This will be fixed in `1.3` by using a different more lax approach to interprocess communication.

**Comment by Wessie** [comment]
Fixed due to the slow move to `jsonrpc`
